docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.8'
+
+services:
+  api:
+    build:
+      context: .
+      dockerfile: api/Dockerfile
+    ports:
+      - "4000:4000"
+    environment:
+      - NODE_ENV=production
+    networks:
+      - 4-growth-docker-network
+
+  client:
+    build:
+      context: .
+      dockerfile: client/Dockerfile
+    ports:
+      - "3000:3000"
+    environment:
+      - NODE_ENV=production
```

Comment:
`next build` (the command you are running behind the scenes while we run `pnpm client:build` already sets the `NODE_ENV` to `production`. This might not be needed.